### DISCUSSION TITLE
Help with FluxAI Code Assist

### DISCUSSION THREAD

**Discussion by dictates** [RunOnFlux/flux#1417] [Q&A]
Hello

Apologies if this is the wrong place. I am on the Supporter plan but hitting limits on the Code Assist.

I have only asked around 10 questions today?

<img width="1019" alt="2024-10-25_13-48-12" src="https://github.com/user-attachments/assets/bf3a7d05-384b-4953-ab4d-d422b5d0f63c">

**Comment by w2vy** [comment]
Dave,

Please open a ticket at https://help.runonflux.io/ and in the ticket include the email address for your account.
Also try again to see if it works now, but please open the ticket either way

Also please let me know the ticket number here and I'll forward it

Tom

**Reply by dictates** [reply]
Thanks Tom

Tried again and still no good. Ticket is 3189.

Dave

**Reply by w2vy** [reply]
Support will be in touch.
